vmware-tanzu/velero
# Add priorityclass to podaction plugin

Adding PriorityClass to podAction of backup and restore plugins.

Fixes #3933 

Testings done:
- Create PriorityClass "high-priority", create Pod with PriorityClassName set to "high-priority".  Trigger Backup ==> check the backup image: backup image does contain the priority-class.json.
- Trigger Restore ==> log show PriorityClass is attempted to restore but skipped because "IncludeClusterResources" was set to "false"
- Delete "high-priority" PriorityClass, trigger Restore with "IncludeClusterResources" set to nil, "high-priority" PriorityClass was restored, Pod also restored successfully.
